lib/gtfs_realtime_viz.ex

```diff
@@ -76,8 +91,35 @@ defmodule GTFSRealtimeViz do
   @spec trainify([Proto.vehicle_position], Proto.vehicle_position_statuses, String.t) :: String.t
   defp trainify(vehicles, status, ascii_train) do
     vehicles
-    |> Enum.filter(& &1.current_status == status)
+    |> vehicles_with_status(status)
     |> Enum.map(& "#{ascii_train} (#{&1.vehicle && &1.vehicle.id})")
     |> Enum.join(",")
   end
+
+  @spec trainify_diff([Proto.vehicle_position], [Proto.vehicle_position], Proto.vehicle_position_statuses, String.t, String.t) :: String.t
+  defp trainify_diff(vehicles_base, vehicles_diff, status, ascii_train_base, ascii_train_diff) do
+    base = vehicles_with_status(vehicles_base, status) |> Enum.map(& &1.vehicle && &1.vehicle.id)
+    diff = vehicles_with_status(vehicles_diff, status) |> Enum.map(& &1.vehicle && &1.vehicle.id)
+
+    unique_base = unique_trains(base, diff, ascii_train_base)
+    unique_diff = unique_trains(diff, base, ascii_train_diff)
+
+    [unique_base, unique_diff]
+    |> List.flatten()
+    |> Enum.map(&span_for_id/1)
+    |> Enum.join(",")
+  end
+
+  defp span_for_id({ascii, id}) do
+    "<span class=\"vehicle-#{id}\" onmouseover=\"highlight(#{id}, 'red')\" onmouseout=\"highlight(#{id}, 'black')\">#{ascii} (#{id})</span>"
```

Comment:
Since we're using `phoenix_html`, can this be done with `content_tag` or the like?